Lib/inspect.py

```diff
@@ -2440,11 +2440,11 @@ def __init__(self, name, kind, *, default=_empty, annotation=_empty):
         if kind not in (_POSITIONAL_ONLY, _POSITIONAL_OR_KEYWORD,
                         _VAR_POSITIONAL, _KEYWORD_ONLY, _VAR_KEYWORD):
             raise ValueError("invalid value for 'Parameter.kind' attribute")
-        self._kind = kind
+        self._kind = _ParameterKind(kind)
 
         if default is not _empty:
-            if kind in (_VAR_POSITIONAL, _VAR_KEYWORD):
-                msg = '{} parameters cannot have default values'.format(kind)
+            if self._kind in (_VAR_POSITIONAL, _VAR_KEYWORD):
+                msg = f'{self._kind!s} cannot have default values'
```

Comment:
Why did you remove 'parameters' word from the message?

Reply:
@1st1 
No reason for it, I rollback it.